### ISSUE TITLE
Invalid syntax

### ISSUE THREAD

**Issue by needforspeed** [blueset/ehForwarderBot#122]
Here is the error message when I run the ehForwardBot:
```
zyc@xubuntu-16:~$ ehforwarderbot
Traceback (most recent call last):
  File "/home/zyc/.local/bin/ehforwarderbot", line 7, in <module>
    from ehforwarderbot.__main__ import main
  File "/home/zyc/.local/lib/python3.5/site-packages/ehforwarderbot/__init__.py", line 3, in <module>
    from .channel import EFBChannel
  File "/home/zyc/.local/lib/python3.5/site-packages/ehforwarderbot/channel.py", line 37
    channel_name: str = "Empty channel"
                ^
SyntaxError: invalid syntax

```

Below is what I installed:

```
zyc@xubuntu-16:~$ pip3 show ehforwarderbot
---
Metadata-Version: 2.0
Name: ehforwarderbot
Version: 2.0.0b10
Summary: An extensible message tunneling chat bot framework.
Home-page: https://github.com/blueset/ehforwarderbot
Author: Eana Hufwe
Author-email: ilove@1a23.com
Installer: pip
License: GPLv3
Location: /home/zyc/.local/lib/python3.5/site-packages
Requires: PyYaml
Classifiers:
  Development Status :: 4 - Beta
  License :: OSI Approved :: GNU General Public License v3 (GPLv3)
  Intended Audience :: Developers
  Intended Audience :: End Users/Desktop
  Programming Language :: Python :: 3 :: Only
  Programming Language :: Python :: 3.6
  Programming Language :: Python :: 3.7
  Topic :: Communications :: Chat
  Topic :: Software Development :: Libraries :: Application Frameworks
  Topic :: Utilities
Entry-points:
  [console_scripts]
  ehforwarderbot = ehforwarderbot.__main__:main
You are using pip version 8.1.1, however version 9.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
zyc@xubuntu-16:~$ pip3 show efb-telegram-master
---
Metadata-Version: 2.0
Name: efb-telegram-master
Version: 2.0.0b10
Summary: Telegram Master Channel for EH Forwarder Bot, based on Telegram Bot API.
Home-page: https://github.com/blueset/efb-telegram-master
Author: Eana Hufwe
Author-email: ilove@1a23.com
Installer: pip
License: GPLv3
Location: /home/zyc/.local/lib/python3.5/site-packages
Requires: retrying, pillow, ehforwarderbot, requests, PyYaml, moviepy, language-tags, python-magic, pydub, python-telegram-bot, peewee
Classifiers:
  Development Status :: 4 - Beta
  License :: OSI Approved :: GNU General Public License v3 (GPLv3)
  Intended Audience :: Developers
  Intended Audience :: End Users/Desktop
  Programming Language :: Python :: 3 :: Only
  Programming Language :: Python :: 3.6
  Programming Language :: Python :: 3.7
  Topic :: Communications :: Chat
  Topic :: Utilities
Entry-points:
  [ehforwarderbot.master]
  blueset.telegram = efb_telegram_master:TelegramChannel
You are using pip version 8.1.1, however version 9.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
zyc@xubuntu-16:~$ pip3 show efb-wechat-slave
---
Metadata-Version: 2.0
Name: efb-wechat-slave
Version: 2.0.0a11
Summary: WeChat Slave Channel for EH Forwarder Bot, based on WeChat Web API.
Home-page: https://github.com/blueset/efb-wechat-slave
Author: Eana Hufwe
Author-email: ilove@1a23.com
Installer: pip
License: GPLv3
Location: /home/zyc/.local/lib/python3.5/site-packages
Requires: pillow, pyqrcode, ehforwarderbot, itchat, xmltodict, PyYaml, python-magic
Classifiers:
  Development Status :: 3 - Alpha
  License :: OSI Approved :: GNU General Public License v3 (GPLv3)
  Intended Audience :: Developers
  Intended Audience :: End Users/Desktop
  Programming Language :: Python :: 3 :: Only
  Programming Language :: Python :: 3.6
  Programming Language :: Python :: 3.7
  Topic :: Communications :: Chat
  Topic :: Utilities
Entry-points:
  [ehforwarderbot.slave]
  blueset.wechat = efb_wechat_slave:WeChatChannel
You are using pip version 8.1.1, however version 9.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Can you please let me know how to correct this issue?

TIA

**Comment by blueset** [comment]
Python 3.6 or higher is required to run EH Forwarder Bot.

**Comment by needforspeed** [comment]
Hi @blueset 

Thanks for the quick response.  It doesn't fully resolve the issue:

```
zyc@xubuntu-16:~$ python --version
Python 3.6.4
zyc@xubuntu-16:~$ python3 --version
Python 3.6.4
```

Here is the new error message
```
zyc@xubuntu-16:~$ ehforwarderbot
Traceback (most recent call last):
  File "/home/zyc/.local/bin/ehforwarderbot", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/ehforwarderbot/__main__.py", line 190, in main
    init()
  File "/usr/local/lib/python3.6/dist-packages/ehforwarderbot/__main__.py", line 74, in init
    conf = config.load_config()
  File "/usr/local/lib/python3.6/dist-packages/ehforwarderbot/config.py", line 33, in load_config
    channel = utils.locate_module(data['master_channel'], 'master')
  File "/usr/local/lib/python3.6/dist-packages/ehforwarderbot/utils.py", line 133, in locate_module
    return i.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.6/dist-packages/efb_telegram_master/__init__.py", line 15, in <module>
    from PIL import Image
  File "/usr/lib/python3/dist-packages/PIL/Image.py", line 66, in <module>
    from PIL import _imaging as core
ImportError: cannot import name '_imaging'
```

**Comment by needforspeed** [comment]
@blueset 
never mind, Pillow related. I fixed it.  You can close the issue now

**Comment by blueset** [comment]
BTW, you can also close an issue if you think it is no longer relevant.

**Comment by FWangZil** [comment]
@needforspeed Excuse me, how do you solve it? I have encountered the same problem.
